code/game/machinery/autolathe/artist_bench.dm

```diff
@@ -175,11 +175,11 @@
 	if(inspiration && user.stats.getPerk(PERK_ARTIST))
 		LStats = inspiration.calculate_statistics()
 
-	var/weight_mechanical = 0 + LStats[STAT_MEC]
-	var/weight_cognition = 0 + LStats[STAT_COG]
+//	var/weight_mechanical = 0 + LStats[STAT_MEC]
+//	var/weight_cognition = 0 + LStats[STAT_COG]
 	var/weight_biology = 0 + LStats[STAT_BIO]
 	var/weight_robustness = 0 + LStats[STAT_ROB]
-	var/weight_toughness = 0 + LStats[STAT_TGH]
+//	var/weight_toughness = 0 + LStats[STAT_TGH]
```

Comment:
What do these changes do? Prevent oddities from affecting MEC/COG/TGH?

Reply:
The artist's bench generates weights for determining the chances of what it will create based on the stats of an oddity that has been placed into it. This in particular removes toughness from the equation as it only affected the chances of getting you the RPG revolvers.

Reply:
28-pack abs means rocket launchers, seems legit